lib/scout_apm/utils/active_record_metric_name.rb

```diff
@@ -121,13 +130,17 @@ def regex_name(sql)
             else
               SELECT_LABEL
             end
-          "#{match[3].classify}/#{operation}"
+          "#{match[3].gsub(/\W/,'').classify}/#{operation}"
```

Comment:
What's the purpose of the new gsub? It strips non-word-chars right?

Reply:
Yes - I saw some queries names with a `)`, for example.

Reply:
Ah, hmm - should we extend the underlying regex for it vs. adding the special case here?